inst/include/interface/rcpp/rcpp_objects/rcpp_population.hpp

```diff
@@ -81,23 +116,38 @@ class PopulationInterface : public PopulationInterfaceBase {
 
     // set relative info
     b0->id = this->id;
```

Comment:
are the names "b0" and "d0" defined to be informative in some way?

Reply:
Currently no. At some point I remember a conversation about changing names so they are more informative, but I don't think this was filed as an issue anywhere.

Reply:
opened issue #384